README.md

```diff
@@ -54,7 +54,31 @@ This solution is designed to be deployed as a Kubernetes DaemonSet to automatica
 3. Once disks are configured, the node taint `disk-unconfigured` is removed
 4. Pods can then be scheduled on the node
 
-### Terraform Example
+It is recommended that any daemonsets required for networking or logs run on other nodes, or be configured to tolerate this taint.
+
+##### GCP note
+In GCP the `konnectivity-agent` pods are needed to retrieve any pod logs.
+
+If those run only on nodes with this taint and they do not tolerate it, all pod logs will be inaccessible until the taint is removed.
+
+In the case of failure of the ephemeral disk setup pods, it may be difficult to debug them, as their logs will be inaccessible.
+
+Configuring the `konnectivity-agent` pods to either tolerate the `disk-unconfigured` taint, or to run on nodes without that taint will allow logs to be accessible as normal. A separate pool of nodes for system daemons without ephemeral disks should work fine.
+
+##### Azure note
+Azure AKS nodes do not support removing taints.
+
+The issue for fixing this was closed (https://github.com/Azure/AKS/issues/2934), and Microsoft does not seem to care that they don't support basic Kubernetes functionality.
```

Comment:
Agreed, but perhaps we shouldn't call'em out like this here. 

Maybe worth noting it in the above line with 
per https://github.com/Azure/AKS/issues/2934.